packages/core/notifications/core-notifications-browser-internal/src/toasts/toasts_api.tsx

```diff
@@ -166,6 +174,11 @@ export class ToastsApi implements IToasts {
    */
   public addError(error: Error, options: ErrorToastOptions) {
     const message = options.toastMessage || error.message;
+    const existingToast = this.getByTitle(options.title);
+    if (existingToast) {
+      // toast with the same title is already displayed, no reason to display it again
+      return existingToast;
+    }
```

Comment:
this is just checking by title if a toast already exists, and then is adding no new one ... I'm aware it would need additional checking. 
* this check could be configureable by ErrorToastOptions
* we could generate a checksum of the error instead which could be used for the check for duplicates
* instead of not adding a new toast, an existing one could be updated, incremented with a counter